Apps/Playground/Android/app/src/main/AndroidManifest.xml

```diff
@@ -33,18 +32,21 @@
       <intent-filter >
         <action android:name="android.intent.action.VIEW" />
         <category android:name="android.intent.category.DEFAULT" />
-        <data android:scheme="file" />
+        <data android:scheme="file"
+            android:host="*" />
```

Comment:
What is this?

Reply:
it's a filter for allowed paths which according to the schema is required